index.js

```diff
@@ -78,6 +79,73 @@ export async function downloadAndExtract(repository, options = Object.create(nul
   return result;
 }
 
+export async function getContributorsLastActivities(owner, repository, options = Object.create(null)) {
+  if (typeof owner !== "string") {
+    throw new TypeError("owner must be a string!");
+  }
+
+  if (typeof repository !== "string") {
+    throw new TypeError("repository must be a string!");
+  }
+
+  const { token } = options;
+
+  const contributorsUrl = new URL(`repos/${owner}/${repository}/contributors`, kGithubApi);
+
+  const { data } = await httpie.get(contributorsUrl, {
+    headers: {
+      "User-Agent": "NodeSecure",
+      Authorization: typeof token === "string" ? `token ${token}` : GITHUB_TOKEN
+    },
+    maxRedirections: 1
+  });
+
+  const lastActivities = new Map();
+  async function hydrateLastActivities(contributor) {
```

Comment:
To increase readability, WDYT about putting `hydrateLastActivities` outside of this function?

Reply:
Is this good?